output/schema/schema.json

```diff
@@ -3074,7 +3074,7 @@
           "stability": "beta"
         }
       },
-      "description": "Creates a connector.",
+      "description": "Create a connector.\n\nConnectors are Elasticsearch integrations that bring content from third-party data sources, which can be deployed on Elastic Cloud or hosted on your own infrastructure.\nNative connectors are a managed service on Elastic Cloud.\nConnector clients are self-managed on your infrastructure.",
```

Comment:
```suggestion
      "description": "Create a connector.\n\nConnectors are Elasticsearch integrations that bring content from third-party data sources, which can be deployed on Elastic Cloud or hosted on your own infrastructure.\nElastic managed connectors (Native connectors) are a managed service on Elastic Cloud.\nSelf-managed connectors (Connector clients) are self-managed on your infrastructure.",
```

Reply:
For future reference, reviewing the output folder is not necessary, since those changes are generated from the other files. When we have automated previews it'll be even more unnecessary, since you will be able to see how it looks in the preview.